go.mod

```diff
@@ -36,10 +36,12 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/gnostic v0.5.7-v3refs // indirect
```

Comment:
this here seems to be the issue why client-go is required v0.24. need to check who pulls this in